packages/firestore-types/index.d.ts

```diff
@@ -123,6 +123,22 @@ export class FirebaseFirestore {
    */
   app: any;
 
+  /**
+   * Re-enables use of the network for this Firestore instance after a prior
+   * call to disableNetwork().
+   * @return A promise that is resolved once the network has been enabled.
```

Comment:
Can you add a newline between the description and the jsdoc annotations for consistency with the rest of this file?

[and also sync this with whatever we end up with for Android]

Reply:
Spaces added, will defer to android review before updating comment.